src/protocols/aeternity/libs/AeAccountHdWallet.ts

```diff
@@ -92,6 +93,37 @@ export class AeAccountHdWallet extends AccountBase {
     );
   }
 
+  override async signMessageJWT(
+    message: object,
+    options: Parameters<AccountBase['signMessageJWT']>[1] & InternalOptions,
+  ): Promise<string> {
+    if (options?.aeppOrigin) {
+      const { checkOrAskPermission } = usePermissions();
+      const permissionGranted = await checkOrAskPermission(
+        METHODS.signMessageJWT,
```

Comment:
Maybe we would need to come up with the new permission for this new method.

Reply:
agree

Reply:
isn't `METHODS.signMessageJWT` already its own permission?

Reply:
No, the check will just return `false` every time we check, so the user would need to confirm every time.

Reply:
can you then help me understand what is needed to be done?

Reply:
As an MVP i would go with asking user every time. It's just something for the future improvements.

Reply:
I mean technically, I'm not very familiar with the wallet implementation, could you help me with that.